nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.travis.yml

```diff
@@ -19,19 +17,17 @@ before_install:
 
 install:
   # Install Nextflow
-  - mkdir /tmp/nextflow
-  - cd /tmp/nextflow
+  - mkdir /tmp/nextflow && cd /tmp/nextflow
```

Comment:
Isn't it easier to read like it was before? I can understand if you think this is neater though.

Reply:
Yeah, it's a bit of a nothing change really. Just figured I'd try to reduce the distance to the interesting testing commands lower down.